io.mrarm.mcpelauncher.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "io.mrarm.mcpelauncher",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.14",
+    "command": "mcpelauncher-ui-qt",
+    "cleanup": [
+        "*.a",
+        "*.la",
+        "/bin/protoc",
+        "/bin/zipcmp",
+        "/bin/zipmerge",
+        "/bin/ziptool",
+        "/bin/captoinfo",
+        "/bin/clear",
+        "/bin/infocmp",
+        "/bin/ncursesw5-config",
+        "/bin/reset",
+        "/bin/tabs",
+        "/bin/tic",
+        "/bin/toe",
+        "/bin/tput",
+        "/bin/tset",
+        "/bin/infotocap",
+        "/lib/libprotobuf-lite*",
+        "/lib/libprotoc*",
+        "/lib/libform*",
+        "/lib/libncurses*",
+        "/lib/libpanel*",
+        "/lib/libformw*",
+        "/lib/libtinfo.so.5",
+        "/lib/terminfo",
+        "/lib/libmenu*"
+    ],
+    "cleanup-commands": [
+        "/app/cleanup-BaseApp.sh"
+    ],
+    "finish-args": [
+        "--socket=x11",
+        "--share=network",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--device=all"
+    ],
+    "modules": [
+        "mcpelauncher-manifest-deps.json",
```

Comment:
Nitpick: storing all modules in the main manifest is usually preferred. The recommended key order is that `sources` is the last as it's not really significant for reading. It's your choice though.

Reply:
> storing all modules in the main manifest is usually preferred.
Ok now is all again in one json file.
source moved to bottom, but only-arches is still below the source.